test/sizes.gzip.json

```diff
@@ -0,0 +1,43 @@
+{
+  "amp.js": "92.33 kB",
```

Comment:
good point. we could keep it around too? i'm perfectly fine just reading the table too. (shouldnt be hard)